ledger/blockqueue.go

```diff
@@ -164,6 +164,22 @@ func (bq *blockQueue) syncer() {
 			bq.mu.Unlock()
 
 			minToSave := bq.l.notifyCommit(committed)
+			var earliest basics.Round
+			const maxDeletionBatchSize = 10_000
+			err = bq.l.blockDBs.Rdb.Atomic(func(ctx context.Context, tx *sql.Tx) error {
+				var err0 error
+				earliest, err0 = blockdb.BlockEarliest(tx)
+				if err0 != nil {
+					bq.l.log.Warnf("blockQueue.syncer: BlockEarliest(): %v", err0)
+				}
+				return err
```

Comment:
If I understand this right, if err0 gets populated but err is nil, earliest will end up defaulting to 0. Doesn't that imply there will be nothing to delete/we'll get stuck on an empty 0-10k delete down below? Wondering if we should skip deletion entirely if we get an error on the BlockEarliest call....

Reply:
yep, a typo here, thanks